x-pack/plugins/ml/public/explorer/explorer_swimlane.js

```diff
@@ -322,6 +263,16 @@ export class ExplorerSwimlane extends React.Component {
     timeBuckets.setInterval(`${stepSecs}s`);
     const xAxisTickFormat = timeBuckets.getScaledDateFormat();
 
+    function cellMouseOverFactory(time, i) {
+      // Don't use an arrow function here because we need access to `this`,
+      // which is where d3 supplies a reference to the corresponding DOM element.
+      return function (lane) {
+        const bucketScore = getBucketScore(lane, time);
+        if (bucketScore === 0) { return; }
```

Comment:
nit, rather than breaking out of this function early, you could invert the check:
```
if (bucketScore !== 0) {
  cellMouseover(this, lane, bucketScore, i, time);
}
```